tests/transforms/test_experimental/test_transform_dispatcher.py

```diff
@@ -174,7 +174,7 @@ def qnode_circuit(a):
         assert not dispatched_transform.is_informative
 
     @pytest.mark.parametrize("valid_transform", valid_transforms)
-    def test_integration_dispatcher_with_valid_transform_decorator(self, valid_transform):
+    def test_integration_dispatcher_with_valid_transform_decorator_partial(self, valid_transform):
```

Comment:
@timmysilv This test already uses `partial`. I added a test for the old syntax.

Reply:
oops, the git diff confused me - the collapsed section made me think the next test was actually this test